### ISSUE TITLE
tzinit (and others) should build also for -m68020/-mcpu=5475

### ISSUE THREAD

**Issue by mikrosk** [freemint/mintlib#60]
Just to have it written down somewhere. Initially found out in https://sourceforge.net/p/freemint/mailman/freemint-discuss/thread/2341352.MzOyzyC30C%40earendil/#msg37888534.

**Comment by th-otto** [comment]
Unlike the bootable builds, there is only one archive for the library, so that would only be possible by dropping the tzinit binary. Maybe install it with a different name?

**Comment by mikrosk** [comment]
Yes, that's what I meant. Just drop it. It's not ideal (much better would be having all three executables in one archive) but if it's tool cumbersome currently, just provide separate `tzinit` etc archives and leave mintlib free of any executable.
